docs/CNAME

```diff
@@ -1 +0,0 @@
-lightwood.io
```

Comment:
Is it okay for this to be gone?

Reply:
nevermind, it's still in the Makefile :+1:

Reply:
I think this is required when connecting GitHub pages to a custom domains?

Reply:
Yeah, but it seems it's being generated when doing `make github` in the `doc_build.yml` file, so it _should_ be okay :thinking: